app/views/global_user_roles/search.html.haml

```diff
@@ -0,0 +1,18 @@
+= content_for :h1 do
+  = t("pages.global_settings")
+
+= content_for :sidebar do
+  = render "admin/shared/sidenav_global", sidenav_tab: "global_user_roles"
+
+%h2= t(".h2")
+= form_tag user_search_results_path, id: "ajax_form" do
```

Comment:
Just want to note for future cleanup I ran into several very similar forms (and this PR is responsible for adding one of them):

* [app/views/account_users/user_search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/account_users/user_search.html.haml)
* [app/views/facility_account_users/user_search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/facility_account_users/user_search.html.haml)
* [app/views/facility_accounts/new_account_user_search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/facility_accounts/new_account_user_search.html.haml)
* [app/views/facility_accounts/search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/facility_accounts/search.html.haml)
* [app/views/facility_accounts/user_search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/facility_accounts/user_search.html.haml)
* [app/views/facility_users/search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/facility_users/search.html.haml)
* [app/views/global_user_roles/search.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/global_user_roles/search.html.haml)
* [app/views/product_users/new.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/product_users/new.html.haml)
* [app/views/user_price_group_members/new.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/user_price_group_members/new.html.haml)
* [app/views/users/_search_form.html.haml](https://github.com/tablexi/nucore-open/tree/21b9c1f28f769060d018848c98edda99798a6c47/app/views/users/_search_form.html.haml)